core/include/scipp/core/element/comparison.h

```diff
@@ -17,37 +17,34 @@
 namespace scipp::core::element {
 
 using is_approx_types_t = arg_list_t<
-    double, float, int64_t, int32_t, std::tuple<double, double, float>,
-    std::tuple<float, float, double>, std::tuple<int64_t, int64_t, double>,
-    std::tuple<int32_t, int32_t, double>, std::tuple<int32_t, int64_t, double>,
-    std::tuple<int64_t, int32_t, double>, std::tuple<int64_t, int32_t, int32_t>,
-    std::tuple<int64_t, int64_t, int32_t>,
+    double, float, int64_t, int32_t, std::tuple<float, float, double>,
+    std::tuple<int64_t, int64_t, double>, std::tuple<int32_t, int32_t, double>,
+    std::tuple<int32_t, int64_t, double>, std::tuple<int64_t, int32_t, double>,
     std::tuple<int64_t, int32_t, int64_t>,
     std::tuple<int32_t, int32_t, int64_t>,
     std::tuple<int32_t, int64_t, int64_t>>;
 
-constexpr auto is_approx_units = [](const units::Unit &x, const units::Unit &y,
-                                    const units::Unit &t) {
+constexpr auto isclose_units = [](const units::Unit &x, const units::Unit &y,
+                                  const units::Unit &t) {
   expect::equals(x, y);
   expect::equals(x, t);
   return units::dimensionless;
 };
 
-struct is_close_flags_t
-    : public transform_flags::expect_all_or_none_have_variance_t,
-      public transform_flags::no_out_variance_t {
+struct isclose_flags_t : public transform_flags::expect_no_variance_arg_t<2>,
+                         public transform_flags::expect_no_variance_arg_t<3> {
```

Comment:
There are only 3 args, what is this flag for? No effect?

Reply:
Correct. Has no effect. Coding error thinking in terms of `atol` and `rotl` i'll fix.